New package: DalitzPlot v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: DalitzPlot
- Repository: https://github.com/junhe1979/DalitzPlot.jl
- Created by: @junhe1979
- Version: v0.1.0
- Commit: 4998ee26de05c961d15fcd99057acb2f89090f1d
- Git reference: HEAD

goerz commented:
See https://github.com/JuliaRegistries/General/pull/98709#issuecomment-1887757463

JuliaHubRegistrator committed:
New package: DalitzPlot v0.1.0

UUID: d7968580-7d6f-4f79-bc09-617baf523457
Repo: https://github.com/junhe1979/DalitzPlot.jl.git
Tree: bbf58cee016bcd4ac5a581e2d4073a5dc118dfbe

Registrator tree SHA: 7242ef64be5953fac9bf8b3efa3a7c4d2d44ca09

goerz commented:
Unblocked, although your README could still use some improvements:

* Installation mentions "ThreebodyProblem.jl", which should be updated
* You have docs/stable and docs/dev badges that are broken. Given that this package is for plotting, I would actually recommend setting up a Documenter-based documentation with some `@example` blocks that show the output this package generates (You could also embed the plot that the README-example generates in the README, as an image)
* Some typos ("partilcls")
* The description is probably good enough for people in the field (which is why I unblocked the registration), but I'd recommend giving a bit more background and explanation in the README and/or a Documenter-based documentation website.
